core/src/main/java/org/apache/iceberg/metrics/ScanMetricsUtil.java

```diff
@@ -31,7 +32,11 @@ public static void indexedDeleteFile(ScanMetrics metrics, DeleteFile deleteFile)
     metrics.indexedDeleteFiles().increment();
 
     if (deleteFile.content() == FileContent.POSITION_DELETES) {
-      metrics.positionalDeleteFiles().increment();
+      if (ContentFileUtil.isDV(deleteFile)) {
```

Comment:
should this be using `ScanTaskUtil.isDV()` now?

Reply:
Replied with context [here](https://github.com/apache/iceberg/pull/11467#discussion_r1829495298).